File: content/snippet/code-signal/arcade/intro/smooth-sailing/common-character-count/common-character-count.js
function commonCharacterCount(s1, s2) {
  let count = 0;
  for (let i = 0; i < s1.length; i++) {
    if (s2.includes(s1[i])) {
      count++;
      s2 = s2.replace(s1[i], '');
    }
  }
  return count;
}

console.log(commonCharacterCount('aabcc', 'adcaa'));

File: content/snippet/code-signal/arcade/intro/smooth-sailing/common-character-count/common-character-count.py
def common_character_count(s1, s2):
    count = 0

    for i in s1:
        for j in range(len(s2)):
            if i == s2[j]:
                count += 1
                s2 = s2[:j] + s2[j+1:]
                break
    return count


print(common_character_count("aabcc", "adcaa"))
